src/components/benchmarks/Benchmarks.tsx

```diff
@@ -0,0 +1,183 @@
+import { useRouter } from "next/router";
+import { useState, MouseEvent } from "react";
+import Stack from "@mui/material/Stack";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import $GoalPage from "@/styles/GoalPage.module.css";
+import $button from "@/components/design_system/button/Button.module.css";
+import Link from "next/link";
+import { Typography } from "@mui/material";
+import Subgoals from "@/components/subgoal/Subgoal";
+import { type Subgoal } from "@/types/global";
+enum selectionValue {
+  ALL,
+  COMPLETE,
+}
+
+interface SelectableTabProps {
+  text: string;
+  value: selectionValue;
+  handleSelect: (e: MouseEvent) => void;
+  currentlySelected: selectionValue;
+}
+
+const SelectableTab = ({
+  text,
+  value,
+  handleSelect,
+  currentlySelected,
+}: SelectableTabProps) => {
+  return (
+    <div
+      className={`${$GoalPage.benchmarksTab}
+        ${value === currentlySelected ? $GoalPage.benchmarksTabSelected : ""} `}
+      onClick={(e) => {
+        handleSelect(e);
+      }}
+    >
+      {text}
+    </div>
+  );
+};
+
+const NoBenchmarksGraphic = ({ blurb }: { blurb?: string }) => {
+  const router = useRouter();
+  return (
+    <Grid
+      sx={{
+        width: "100%",
+        height: "100%",
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "center",
+        alignItems: "center",
+        background: "white",
+      }}
+      item
+    >
+      <svg
```

Comment:
Looking at the design, the design version of the image seems to have it shifted slightly to the left, so that the box part of it seems centered rather than the whole image. I'm not sure what the best way of doing that would be. I also noticed that there are some similar pngs in public/image, so perhaps it would be better to export the png from figma, stash it with those others and then import it into here in place of the svg?

Reply:
I adjusted the margins/padding such that the box that the box is properly centered and more in line with the original figma design.

Reply:
Looks much better